linera-chain/src/chain.rs

```diff
@@ -1167,8 +1157,15 @@ where
                         message.grant == Amount::ZERO,
                         ChainError::GrantUseOnBroadcast
                     );
+                    let GenericApplicationId::User(application_id) =
```

Comment:
Is `GenericApplicationId::System` used anywhere else now?

Reply:
Yes, in events.